app/assets/stylesheets/global.sass

```diff
@@ -329,3 +329,7 @@ h1.header, h2.header
   h5
     font-style: italic
     margin-bottom: 0
+
+.modal
+  h2
+    display: inline
```

Comment:
I may be missing something, but can you explain why this is necessary?

Reply:
This is necessary because the default display of a h2 element is a block. In this case, the modal title and the close button get misaligned.